addons/account/views/account_view.xml

```diff
@@ -1077,7 +1077,7 @@
             <field name="model">account.tax</field>
             <field name="arch" type="xml">
                 <tree string="Account Tax">
-                    <field name="name"/>
+                    <field name="display_name" string="name"/>
```

Comment:
Why only in this tree view, and not the other one?

Reply:
@oco-odoo , 
when we use the `search more` option of many2one field dropdown at that time it will open this tree view and on that tree view we want to display the type of tax(i.e: 5% Tax(sales)) 
so to display type of tax based on context `append_type_to_tax_name` we use the `display_name` instead of `name`. while on other views we do not want to display it so keep that as it is. Thanks

Reply:
Alright, seems legit.